samples/public/BlankUwpNet9App/BlankUwpNet9App.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net9.0-windows10.0.26100.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <WindowsSdkPackageVersion>10.0.26100.57</WindowsSdkPackageVersion>
+    <Platforms>x86;x64;arm64</Platforms>
+    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+    <PublishProfile>win-$(Platform).pubxml</PublishProfile>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <PublishAot>true</PublishAot>
+    <UseUwp>true</UseUwp>
+    <DisableRuntimeMarshalling>true</DisableRuntimeMarshalling>
+    <EnableMsixTooling>true</EnableMsixTooling>
+	  <UnitTestPlatformVersion Condition="'$(UnitTestPlatformVersion)' == ''">$(VisualStudioVersion)</UnitTestPlatformVersion>
+
+    <!-- Set this property to override the version of the MSIX package to use -->
+    <MicrosoftWindowsSDKBuildToolsMSIXPackageVersion>1.0.20241009.2</MicrosoftWindowsSDKBuildToolsMSIXPackageVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+	  <ProjectCapability Include="TestContainer" />
+	  <PackageReference Include="Microsoft.TestPlatform" Version="17.13.0-dev" />
```

Comment:
Is there another PR that touches VSTest? If so, we will need to later update the sample with a version available on NuGet instead of a locally built version

Reply:
Yea I had some changes in vstest to remove subscribing to the process, which doesn't work in UWP. @Evangelink is working on a PR for that. I'll circle back and update these to the public version when that gets published.

Reply:
@adstep I don't recall if you did some changes to VSTest or not.

Reply:
@Evangelink I had to remove UnitTestClient from subscribing to [Parent process' ExitCallback](https://github.com/microsoft/vstest/blob/bcef12d909709f13027afe1557c724f11bc8df05/src/testhost.x86/DefaultEngineInvoker.cs#L206-L238). ProcessHelper throws in the UWP setting because it executes in a sandbox that doesn't have permissions to access process info. 

The old VSTest framework that works with legacy UWP just had ProcessHelper as  a complete no-op for all the methods.